documentation/general/general_setup.md

```diff
@@ -1,263 +1,230 @@
-prerequisites are required to get started with your contributions.
-
-| Requirement        | Docs | Ingestion server | API | Frontend       | Management |
-| ------------------ | ---- | ---------------- | --- | -------------- | ---------- |
-| [Git](#git)        | ✅   | ✅               | ✅  | ✅             | ✅         |
-| [`just`](#just)    | ✅   | ✅               | ✅  | ✅             | ✅         |
-| [Python](#python)  | ✅   | ➖               | ➖  | ➖             | ✅         |
-| [Node.js](#nodejs) | ➖   | ➖               | ➖  | ✅             | ✅         |
-| [Docker](#docker)  | ➖   | ✅               | ✅  | ❔[^analytics] | ➖         |
-
-Here ✅ means required, ➖ means not required and ❔ means conditionally
-required.
+## Dependencies
 
-[^analytics]: This is required to run analytics, not required otherwise.
+You must have available on your computer (referred to as the "host system") the
+following:
 
-## Required setup
+- bash
+- git
+- Docker (or a compatible container runtime)
 
-The following setup steps are needed to set up a local copy of Openverse and do
-not prepare you to contribute code to the project.
+On macOS and most popular Linux distributions, bash and git are installed by
+default. Most contributors will only need to install Docker or a compatible
+container runtime to use `ov`.
 
-### Git
+[Follow Docker's instructions to install Docker on your host system](https://docs.docker.com/engine/install/).
 
-Openverse is Git-tracked. To clone Openverse locally, you will need to install
-`git`. Most Linux distributions and macOS will already have this installed, even
-if it's not always the latest version. To know if you have Git installed, run
-the following command.
+```{tip}
+For Openverse, you'll only need the container runtime, and do not necessarily
+need to install additional tools like Docker compose, buildx, and so forth.
 
-```bash
-git --version
+While Openverse relies on these tools, they are all installed in the `ov` container,
+and therefore do not need to be available on your host system if you won't use
+them otherwise.
 ```
 
-If you see `git version x.y.z`, you have Git installed. If you see an error, you
-need to install it by following the
-[official instructions](https://git-scm.com/downloads).
-
-### `just`
+## Up and running
 
-We use `just` as our command runner. It makes it easier to run cumbersome
-commands that are generally needed a lot during development, like bringing up
-our backend services or linting the codebase.
+1. Clone the GitHub repository:
 
-`just` can be [installed](https://github.com/casey/just#installation) for a host
-of operating systems via their respective
-[package managers](https://github.com/casey/just#packages) or using
-[pre-built binaries](https://github.com/casey/just#pre-built-binaries) available
-for some operating systems.
+```shell
+git clone --filter=blob:none https://github.com/WordPress/openverse.git # or your fork
```

Comment:
Do you think we should explain `--filter=blob:none` at all? It might not be important enough to warrant the previous, verbose description:

```
\```{note}
   We recommend cloning with the `--filter=blob:none` flag as it dramatically
   reduces the filesize and download time by creating a "blobless clone".
   You can learn more about these [here](https://gist.github.com/leereilly/1f4ea46a01618b6e34ead76f75d0784b).
\```
```

Reply:
Yes, definitely worth adding context.